src/Scan/Scans.ts

```diff
@@ -50,7 +50,8 @@ export enum TestType {
   WORDPRESS = 'wordpress',
   XPATHI = 'xpathi',
   XSS = 'xss',
-  XXE = 'xxe'
+  XXE = 'xxe',
+  GRAPHQL_INTROSPECTION = 'graphql_introspection'
```

Comment:
I assume you need to follow the alphabetical order convention

Reply:
yep, will be good to follow